packages/web/src/components/CheckboxField/_CheckboxField.scss

```diff
@@ -0,0 +1,90 @@
+@use 'theme';
+@use '../../tools/typography';
+
+.CheckboxField {
+    max-width: 100%;
+}
+
+.CheckboxField__label {
+    @include typography.generate(theme.$label-typography);
+
+    margin-bottom: theme.$spacing;
+    vertical-align: top;
+    color: theme.$label-color-default;
+}
+
+.CheckboxField__label--hidden {
+    position: absolute;
+    overflow: hidden;
+    width: 1px;
+    height: 1px;
+    padding: 0;
+    margin: -1px;
+    clip: rect(0, 0, 0, 0);
+    white-space: nowrap;
+    border: 0;
+}
+
+.CheckboxField__label--required::after {
+    content: '*';
+    color: theme.$label-required-color;
+}
+
+.CheckboxField__input {
+    appearance: none;
+    width: theme.$input-width;
+    height: theme.$input-width;
+    margin-right: 5px;
```

Comment:
The margins need improvement. They are not according to design and they depend on native browser spacing. Please fix this (after placing the input inside the label).

Reply:
Well, I was missing an additional 1px. But not sure if I must use some kind of CSS reset. 🤔